com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Main.md

```diff
@@ -1,15 +1,15 @@
 # Post-processing in the High Definition Render Pipeline
 
-The High Definition Render Pipeline (HDRP) includes its own purpose-built implementation for [post-processing](https://docs.unity3d.com/Manual/PostProcessingOverview.html). This is built into HDRP, so you do not need to install any other package.
+The High Definition Render Pipeline (HDRP) includes its own purpose-built implementation for [post-processing](https://docs.unity3d.com/Manual/PostProcessingOverview.html). This is built into HDRP, so you don't need to install any other package.
 
-This implementation uses the [Volume](Volumes.md) system. You add post-processing effects to your Camera in the same way you add any other [Volume Override](Volume-Components.md). Note that HDRP already enables some post-processing effects in the [HDRP Global Settings](Default-Settings-Window.md), go to Section **Volume Profiles**.
+This implementation uses the [Volume](Volumes.md) system. You add post-processing effects to your Camera in the same way you add any other [Volume Override](Volume-Components.md). 
 
-The images below show a Scene with and without HDRP post-processing.
+**Note**: HDRP already enables some post-processing effects in the [HDRP Global Settings](Default-Settings-Window.md), go to Section **Volume Profiles**.
```

Comment:
If you mean the Volume profile section in this doc, link to it. 
To do that, use a header link and then a md link with the format [Header title](#header-title).
(I can find an example of a header link if you need one.)

Reply:
Also I'd change the phrasing here to say some things are enabled by default (I know it sounds like passive voice but checkboxes are the exception): 

"Some post-processing effects are enabled by default in the [HDRP Global Settings](Default-Settings-Window.md)..."